table/block_based/block_based_table_builder.cc

```diff
@@ -299,12 +300,13 @@ struct BlockBasedTableBuilder::Rep {
   PartitionedIndexBuilder* p_index_builder_ = nullptr;
 
   std::string last_key;
+  const Slice* first_key_in_next_block_ptr = nullptr;
```

Comment:
Is it possible to make it std::unique_ptr<const Slice> so that we don't have to do the cleaning up with delete?

Reply:
Fixed.